contributors/index.html

```diff
@@ -154,7 +154,7 @@ <h5 class="subheader">The Bio-Formats project has been fortunate to receive assi
             </div>
 
             <div class="large-expand columns">
-            Barry DeZonia <a target="_blank" href="http://loci.wisc.edu/people/barry-dezonia">UW-Madison LOCI</a>
+            Barry DeZonia <a target="_blank" href="http://loci.wisc.edu/">UW-Madison LOCI</a>
```

Comment:
I imagine it's possible to find new landing pages for these (github/linkedin). Unclear if that's something we want to invest in.

Reply:
@mtbc has been in touch with the LOCI group but I am not sure whether a clear remapping has emerged yet or is upcoming.

Reply:
Silence since early this month. I can re-poke if desired.